html/webappapis/scripting/processing-model-2/unhandled-promise-rejections/promise-rejection-events-onerror.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<link rel="help" href="https://html.spec.whatwg.org/#unhandled-promise-rejections">
+<script>
+'use strict';
+setup({
+  allow_uncaught_exception: true
+});
+async_test(function(t) {
+  var e = new Error('e');
+  var e2 = new Error('e2');
+  window.onerror = function (msg, url, line, col, error) {
+    t.step(function() {
+      assert_equals(msg, 'Uncaught Error: e2');
+      assert_equals(error, e2);
+    });
+    t.done();
+  };
+  window.onrejectionhandled = function() {
+    throw e2;
+  };
+  var p = Promise.reject(e);
+  setTimeout(function() {
+    try {
+      p.catch(function() {});
+    } catch (e) {
+      assert_unreached('attaching a handler should not throw an error');
+    }
+  }, 1);
```

Comment:
No; the seqence is Promise.reject(e); setTimeout 1; p.catch -> causes onrejectionhandled; throw e2 -> causes onerror; t.done().